packages/nitrogen/src/nitrogen.ts

```diff
@@ -101,7 +103,8 @@ export async function runNitrogen({
           continue
         }
 
-        const platforms = Object.keys(platformSpec) as Platform[]
+        platforms = Object.keys(platformSpec) as Platform[]
```

Comment:
This overwrites `platforms` each time instead of appending to it.

Reply:
Instead of lifting out `platforms` here, let's create a new value in line 82 called `usedPlatforms` and we additionally append to it.

Reply:
addressed the issue above